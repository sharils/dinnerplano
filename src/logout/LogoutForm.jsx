import React from "react";
import { propTypes, reduxForm } from "redux-form";

function LogoutForm(props) {
  return (
    <form onSubmit={props.handleSubmit}>
      <button type="submit" disabled={props.submitting}>
        Logout
      </button>
      {props.error}
    </form>
  );
}
LogoutForm.propTypes = propTypes;

export default reduxForm({
  form: "LogoutForm"
})(LogoutForm);
