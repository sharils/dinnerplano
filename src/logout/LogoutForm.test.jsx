import React from "react";
import renderer from "react-test-renderer";
import Provizanto from "../../test/util/Provizanto";
import LogoutForm from "./LogoutForm";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Provizanto>
        <LogoutForm onSubmit={() => {}} />
      </Provizanto>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
