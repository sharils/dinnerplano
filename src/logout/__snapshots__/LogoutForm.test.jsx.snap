// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <button
    disabled={false}
    type="submit"
  >
    Logout
  </button>
</form>
`;
